examples/mark_bad_channels.py

```diff
@@ -1,6 +1,6 @@
 """
 ===============================================
-10. Changing which channels are marked as "bad"
+03. Changing which channels are marked as "bad"
```

Comment:
I just noticed L28 in this example. That's bad, you're setting the MNE log level silently whenever user runs the example.